src/Conditionals.elm

```diff
@@ -0,0 +1,489 @@
+module Conditionals exposing (..)
+
+
+calculateGrade : Int -> String
+calculateGrade score =
+    if score >= 100 && score <= 0 then
+        "Invalide Score"
+        -- If score is greater than or equal to 100 and score
+        -- is less than or equal 0  "Invalide Score" will be assign as the grade.
+
+    else if score >= 90 then
+        "A"
+        --If score is greater than or equal 90 "A"  will be assign as the grade.
+
+    else if score >= 80 then
+        "B"
+        --If score is greater than or equal 80 "B" will be assign as the grade.
+
+    else if score >= 70 then
+        "C"
+        --If score is greater than or equal 70 "C" will be assign as the grade.
+
+    else if score >= 60 then
+        "D"
+        --If the score greater than or equal 60 "D" will be assign as the grade.
+
+    else
+        "F"
+
+
+
+--Anything else from 59 to 0 should return "F"
+-- This function  takes an Int and returns a string,
+-- Then classify ages into different groups.
+
+
+categorizeCharacter : Char -> String
+categorizeCharacter char =
+    --This line checks if the character is a vowel for both uppercase or lowercase
+    if char == 'A' || char == 'E' || char == 'I' || char == 'O' || char == 'U' || char == 'a' || char == 'e' || char == 'i' || char == 'o' || char == 'u' then
+        "Vowel"
+
+    else if Char.isDigit char then
+        "Digit"
+        --This line Checks if the character is a digit
+
+    else if (char >= 'a' && char <= 'z') || (char >= 'A' && char <= 'Z') then
+        "Consonant"
+        --This line checks if character is a consonant for both uppercase or lowercase
+
+    else
+        "Unknown"
+
+
+
+--Any other special character like '@' is categorize as "Unknown".
+
+
+ageClassifier : Int -> String
+ageClassifier age =
+    if age < 0 || age > 1000 then
+        "Invalide age"
+        -- Check if age is less than 0 then age should return ("Invalide age")
+
+    else if age <= 12 then
+        "Child"
+        -- Check if age is between 0 to 12 range then ("Child")
+
+    else if age <= 19 then
+        "Teenager"
+        -- Check if age is between 13 to 19 range then ("Teenager")
+
+    else if age <= 64 then
+        "Adult"
+        --Check if age is between 20 to 64 range then ("Adult")
+
+    else if age <= 122 then
+        "Senior"
+        --Check if age is between 65 to 122 range then ("Senior")
+
+    else if age <= 969 then
+        "Guinness World Records, let's goooooooo!"
+        --Check if age is between 123 to 969 range then
+        --("Guinness World Records, let's goooooooo!")
+
+    else
+        "Methuselah?!"
+
+
+
+--Check if age is between 968 to 1000 range then
+--("Methuselah?!")
```

Comment:
Whoops.

I think most of your comments are leaving where they're supposed to be.

To comment an if else expression, try something like this:

```elm
if <blah blah blah> then
    -- comments here
    -- more
else
    -- another set of comments here
    -- more
```

Also, please put a space between the `--` and the sentence that follows it.